integration/pvtdata/pvtdata_test.go

```diff
@@ -103,6 +103,64 @@ var _ bool = Describe("PrivateData", func() {
 			assertPvtdataPresencePerCollectionConfig1(network, testChaincode.Name, "marble1")
 		})
 	})
+	Describe("Implicit Collection Config", func() {
+		It("Disseminates private data via the implicit collection without erroring", func() {
+			By("setting up the network")
+			network = initThreeOrgsSetup()
+			newLifecycleChaincode := nwo.Chaincode{
+				Name:              "marblesp",
+				Version:           "1.0",
+				Path:              components.Build("github.com/hyperledger/fabric/integration/chaincode/marbles_private/cmd"),
+				Lang:              "binary",
+				PackageFile:       filepath.Join(network.RootDir, "marbles-pvtdata.tar.gz"),
+				Label:             "marbles-private-20",
+				SignaturePolicy:   `OR ('Org1MSP.member','Org2MSP.member', 'Org3MSP.member')`,
+				CollectionsConfig: collectionConfig("collections_config1.json"),
+				Sequence:          "1",
+			}
+
+			By("starting the network")
+			process, orderer = startNetwork(network)
+
+			By("enabling V2_0 capabilities")
+			nwo.EnableCapabilities(network, channelID, "Application", "V2_0", orderer, network.Peers...)
+
+			By("setting the PrivateDataImplicitCollection")
+			Expect(network.Peers).NotTo(BeEmpty())
+			config := nwo.GetConfig(network, network.Peers[0], orderer, channelID)
+			updatedConfig := proto.Clone(config).(*cb.Config)
+
+			for _, o := range network.Organizations {
+				if o.Name != "OrdererOrg" {
+					updatedConfig.ChannelGroup.Groups["Application"].Groups[o.Name].Values["PrivateDataImplicitCollection"] = &cb.ConfigValue{
+						ModPolicy: "Admins",
+						Value: protoutil.MarshalOrPanic(
+							&pb.PrivateDataImplicitCollection{
+								RequiredPeerCount: 0,
+								MaxPeerCount:      1,
+								BlockToLive:       10,
+								MemberOnlyRead:    true,
+								MemberOnlyWrite:   true,
+							},
+						),
+					}
+				}
+			}
+
+			nwo.UpdateConfig(network, orderer, channelID, config, updatedConfig, false, network.Peers[0], network.Peers...)
+
+			By("deploying new lifecycle chaincode and adding marble1")
+			testChaincode := chaincode{
+				Chaincode: newLifecycleChaincode,
+				isLegacy:  false,
+			}
```

Comment:
What are we actually testing here? Would any of this fail if line 150 were commented out?

Reply:
I guess this test was more validating as a sanity check if the capability was properly set. Now that it's a channel capability which is defaulted to have V2_0 support in the integration test, none of this should fail anyways. In all honesty it would probably make more sense to add some implicit collection read/write support in the test chaincode itself to actually verify if dissemination works. That might be worth exploring on a separate issue though since this one is already growing fairly large.